package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "simple-server-node",
+  "version": "1.0.0",
+  "description": "A simple Expressjs Web server on balena",
+  "main": "server.js",
+  "scripts": {
+    "build": "tsc",
+    "start": "node dist/server.js"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/balena-io-projects/simple-server-node.git"
+  },
+  "keywords": [
+    "webserver",
+    "nodejs",
+    "expressjs"
+  ],
+  "author": "Shaun Mulligan <shaun@balena.io>",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/balena-io-projects/simple-server-node/issues"
+  },
+  "homepage": "https://github.com/balena-io-projects/simple-server-node#readme",
```

Comment:
nit:  I feel like some of this metadata needs to change?

- [ ] The git URLs are referencing the original project.
- [ ] Should the keywords include 'typescript'?
- [ ] Name and description
- [ ] Author

Reply:
Good spot @toochevere, done. New urls reference the examples repo, where this one hopefully will move to later.